README.adoc

```diff
@@ -336,7 +335,7 @@ The Maven `pom.xml` file contains a configuration for the [hotspot=maven-depende
 
 Also, add and configure the [hotspot=failsafe file=0]`maven-failsafe-plugin` plugin, so that the integration test can be run by the `mvn verify` command.
 
-As you started Open Liberty in dev mode, all the changes were automatically picked up. You can run the tests by pressing the `enter/return` key from the command-line session where you started dev mode. You will see the following output:
+As you started Open Liberty in dev mode, all the changes were automatically picked up. You can run the tests by pressing the `enter/return` key from the command-line session where you started dev mode. You see the following output:
```

Comment:
As you started Open Liberty in dev mode
--->
When you started Open Liberty in dev mode